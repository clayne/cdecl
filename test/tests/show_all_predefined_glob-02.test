cdecl @ @ -xc++ @ show all predefined *::atomic* as typedef @ 0
