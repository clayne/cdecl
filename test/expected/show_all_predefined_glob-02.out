namespace std { struct atomic_bool; }
namespace std { struct atomic_char; }
namespace std { struct atomic_char16_t; }
namespace std { struct atomic_char32_t; }
namespace std { struct atomic_char8_t; }
namespace std { struct atomic_flag; }
namespace std { struct atomic_int; }
namespace std { struct atomic_int16_t; }
namespace std { struct atomic_int32_t; }
namespace std { struct atomic_int64_t; }
namespace std { struct atomic_int8_t; }
namespace std { struct atomic_int_fast16_t; }
namespace std { struct atomic_int_fast32_t; }
namespace std { struct atomic_int_fast64_t; }
namespace std { struct atomic_int_fast8_t; }
namespace std { struct atomic_int_least16_t; }
namespace std { struct atomic_int_least32_t; }
namespace std { struct atomic_int_least64_t; }
namespace std { struct atomic_int_least8_t; }
namespace std { struct atomic_intmax_t; }
namespace std { struct atomic_intptr_t; }
namespace std { struct atomic_llong; }
namespace std { struct atomic_long; }
namespace std { struct atomic_ptrdiff_t; }
namespace std { struct atomic_schar; }
namespace std { struct atomic_short; }
namespace std { struct atomic_signed_lock_free; }
namespace std { struct atomic_size_t; }
namespace std { struct atomic_uchar; }
namespace std { struct atomic_uint; }
namespace std { struct atomic_uint16_t; }
namespace std { struct atomic_uint32_t; }
namespace std { struct atomic_uint64_t; }
namespace std { struct atomic_uint8_t; }
namespace std { struct atomic_uint_fast16_t; }
namespace std { struct atomic_uint_fast32_t; }
namespace std { struct atomic_uint_fast64_t; }
namespace std { struct atomic_uint_fast8_t; }
namespace std { struct atomic_uint_least16_t; }
namespace std { struct atomic_uint_least32_t; }
namespace std { struct atomic_uint_least64_t; }
namespace std { struct atomic_uint_least8_t; }
namespace std { struct atomic_uintmax_t; }
namespace std { struct atomic_uintptr_t; }
namespace std { struct atomic_ullong; }
namespace std { struct atomic_ulong; }
namespace std { struct atomic_unsigned_lock_free; }
namespace std { struct atomic_ushort; }
namespace std { struct atomic_wchar_t; }
